sanghyeok/LV_1/두-정수-사이의-합.js

```diff
@@ -0,0 +1,5 @@
+function solution(a, b) {
+    let sum = 0;
+    for(let i = Math.min(a, b); i < Math.max(a, b) + 1; i++) sum += i;
+    return sum;
+}
```

Comment:
```suggestion
function solution(a, b) {
    let sum = 0;
    for(let i = Math.min(a, b); i < Math.max(a, b) + 1; i++) sum += i;
    return sum;
}
```
for문을 줄여서 작성한 상혁님 코드 잘 보고 갑니다!